tailwind.config.d.ts

```diff
@@ -0,0 +1,796 @@
+// #region Tailwind default config
+type DefaultConfig = {
```

Comment:
Had to manually define the type of the default tailwind config because there is no way to infer the types from a .js file while importing it as a module 

https://github.com/tailwindlabs/tailwindcss/discussions/1077#discussioncomment-56079

Reply:
Never mind............ I can totally eliminate the need for the `DefaultConfig` type by using a generic which can infer the type from the implicit parameter..

```ts
preset<T extends { purge: any }>(tailwindConfig: T): T
```